packages/components/modules/profiles/graphql/queries/ProfilesList.ts

```diff
@@ -0,0 +1,11 @@
+import { graphql } from 'react-relay'
+
+export const ProfilesListQuery = graphql`
+  query ProfilesListQuery {
+    me {
+      profiles {
```

Comment:
Although we not handling pagination on this story, it would be nice if we had paginated query already... @nossila don't we have any examples of that query on AIRTAB?

Reply:
I would need to cover that in the BE as well. I aligned with @nossila and IC, about that. We will need to create a separated story for handling the pagination of profiles.
That wasn't covered in the AC and since I'm already 200% over budget I think we can keep it like that for now and wait this new story.

Reply:
As aligned, we will implement the pagination in a new story